sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1681,30 +1708,156 @@ public PDone apply(PCollection<TableRow> input) {
         if (Strings.isNullOrEmpty(table.getProjectId())) {
           table.setProjectId(options.getProject());
         }
-        String jobIdToken = randomUUIDString();
+        final String jobIdToken = "beam_job_" + randomUUIDString();
```

Comment:
is this `final` necessary? Harmless, but I normally don't add them to scoped local variables unless they're required.